building.html

```diff
@@ -69,24 +71,20 @@ <h3 id="dependencies">Library Dependencies</h3>
 
       <h3 id="building-windows">Building on Windows</h3>
 
-      <p>Compiling on Windows requires Visual Studio. We recommend <a href="https://visualstudio.microsoft.com/downloads/">Visual Studio 2019 Community Edition</a>, which is available as a free download. Install the “Desktop development with C++” workload.</p>
+      <p>Compiling on Windows requires Visual Studio. We recommend <a href="https://visualstudio.microsoft.com/downloads/">Visual Studio 2022 Community Edition</a>, which is available as a free download. Install the “Desktop development with C++” workload.</p>
 
       <ol>
         <li>
-          <p>Clone the repository from GitHub, including all submodules, using a git client. On the command line, this can be done by executing the following:<br>
-            <code>git clone --recurse-submodules https://github.com/H-uru/Plasma.git</code></p>
-
-          <p class="infonote" data-infotype="warning"><b>Note:</b> We recommend using <a href="https://git-scm.com/download/win">Git for Windows</a>. Using the git program provided by Cygwin will not set permissions properly on some files and result in build failures.</p>
-
-          <p class="infonote" data-infotype="warning"><b>Note:</b> Some dependencies may fail to build if the path to the code contains spaces.</p>
```

Comment:
Was this removed on purpose? Last I checked we still have at least one autoconf-based dependency, which won't work properly with spaces in the path.

Reply:
oh, that is a good catch, I'll add that back to the step about cloning to a folder